wfe2/verify_test.go

```diff
@@ -1495,6 +1496,38 @@ func TestValidPOSTForAccountSwappedKey(t *testing.T) {
 	test.AssertEquals(t, prob.Detail, "JWS verification error")
 }
 
+func TestValidSelfAuthenticatedPOSTGoodKeyErrors(t *testing.T) {
```

Comment:
The first case here goes
1) make the request
2) make the timeout func
3) make the policy
4) run the code under test

The second case here moves step (1) to below step (3). I don't think there's a right or wrong order, but they should be in the same order.